### PR TITLE
User controller

### PR DESCRIPTION
**Pull request by darwinaraqueo-zz**
**Summary**
Modification for user controller, this allow have rest controllers for use in cruds

**Trello Card**
[Go!](https://trello.com/c/GjwPVfrj/22-crear-user-controller-con-las-operaciones-crud-de-user)

### CODE REVIEW

File: src/main/java/wolox/training/controller/BookController.java [hunk 1]
```diff
@@ -18,6 +19,7 @@
 import wolox.training.repositories.BookRepository;
 
 @Controller
```
Comment (zeminlu) [inline src/main/java/wolox/training/controller/BookController.java]: We use RestController for an REST API app.

File: src/main/java/wolox/training/controller/UserController.java [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+import org.springframework.web.bind.annotation.ResponseStatus;
+import wolox.training.exceptions.BookNotFoundException;
+import wolox.training.exceptions.UserIdMismatchException;
+import wolox.training.exceptions.UserNotFoundException;
+import wolox.training.models.Book;
+import wolox.training.models.User;
+import wolox.training.repositories.BookRepository;
+import wolox.training.repositories.UserRepository;
+
+@Controller
```
Comment (zeminlu) [inline src/main/java/wolox/training/controller/UserController.java]: We use RestController for an REST API app.

File: src/main/java/wolox/training/exceptions/UserNotFoundException.java [hunk 3]
```diff
@@ -0,0 +1,11 @@
+package wolox.training.exceptions;
+
+public class UserNotFoundException extends RuntimeException {
```
Comment (zeminlu) [inline src/main/java/wolox/training/exceptions/UserNotFoundException.java:7]: This would be a not found http return status code.

**Review by zeminlu — CHANGES_REQUESTED** [review]
Looks good, check my comments!

File: src/main/java/wolox/training/controller/BookController.java (hunk 1 above)
Comment (darwinaraqueo-zz) [reply src/main/java/wolox/training/controller/BookController.java]: ready

**Review by zeminlu — APPROVED** [review]

**Pull request merged**